openvdb_cmd/vdb_tool/src/main.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+////////////////////////////////////////////////////////////////////////////////
+///
+/// @author Ken Museth
+///
+/// @file main.cpp
+///
+/// @brief One-stop command-line tool for printing, converting, processing and
+///        and rendering of VDB grids.
+///
+////////////////////////////////////////////////////////////////////////////////
+
+
+#include "Tool.h"
+
+int main(int argc, char *argv[])
+{
+    int exitStatus = EXIT_SUCCESS;
+
+    try {
+
+        openvdb::vdb_tool::Tool tool(argc, argv);
+        tool.run();
+
+    } catch (const std::exception& e) {
+
+        OPENVDB_LOG_FATAL(std::string("Tool: ")+e.what());
+        exitStatus = EXIT_FAILURE;
+
+    } catch (...) {
+
+        OPENVDB_LOG_FATAL("Tool: exception of unknown type caught");
+        exitStatus = EXIT_FAILURE;
+        std::unexpected();
```

Comment:
I got the following error trying to compile vdb_tool on Visual Studio 2019:
C:\Users\apradhana\dev\openvdb_bkup\openvdb_cmd\vdb_tool\src\main.cpp(36,14): error C2039: 'unexpected': is not a membe
r of 'std' [C:\Users\apradhana\dev\openvdb_bkup\build\openvdb_cmd\vdb_tool\vdb_tool.vcxproj]
Seems, from (https://en.cppreference.com/w/cpp/error/unexpected), that this function is deprecated in C++17.

Reply:
replaced "std::unexpected();" with "std::terminate"